src/main/java/software/amazon/cloudformation/proxy/AmazonWebServicesClientProxy.java

```diff
@@ -443,9 +444,24 @@ public Logger getLogger() {
         request.setRequestCredentialsProvider(v1CredentialsProvider);
 
         try {
-            ResultT respose = requestFunction.apply(request);
-            logRequestMetadata(request, respose);
-            return respose;
+            ResultT response = requestFunction.apply(request);
+            logRequestMetadata(request, response);
+            return response;
+        } catch (final Throwable e) {
+            loggerProxy.log(String.format("Failed to execute remote function: {%s}", e.getMessage()));
+            throw e;
```

Comment:
do we want to wrap it in CFN exception? `CfnInvalidRequest` maybe - @wbingli

Reply:
i think we just want to return whatever the sdk returns here so the provider knows what the exception of the sdk was - Cfn prefixed exceptions are for the providers to determine how they are applied. 

If you look down in the diff this is what we were doing previously

Reply:
@jotompki agreed, the mapping will happen inside call chain